### ISSUE TITLE
Learner view: in a lesson, independent practice quizzes can be taken, submitted, and the report can be seen

### ISSUE THREAD

**Issue by sairina** [learningequality/kolibri#8508]
### User stories for this issue:

> As a learner, I want to be able to select an independent practice quiz that was assigned to me in a lesson so that I can take it, submit it, earn points, and view the report.

|Frame|Explanation|
|--|--|
|![Screen Shot 2021-10-20 at 8 23 57 PM](https://user-images.githubusercontent.com/13563002/138207611-12724656-27fb-496b-af77-09d0df256a86.png)|(a) Independent practice quiz can be found in a lesson|
|![Screen Shot 2021-10-25 at 10.00.43 PM.png](https://images.zenhubusercontent.com/6050ea86e1747f15de7d8759/2577c84b-d96f-4d64-bf08-e661627ce462)|(b) Alternative way in which a practice quiz can be accessed when in a lesson - when learner finishes exercise and a practice quiz is next in the lesson and the learner clicks on the next item|
|![Screen Shot 2021-10-20 at 8 24 24 PM](https://user-images.githubusercontent.com/13563002/138207615-407ceac1-505c-4a11-b043-fce00235cfe9.png)|Learner is engaging with a practice quiz and can submit|
|![Screen Shot 2021-10-20 at 8 24 40 PM](https://user-images.githubusercontent.com/13563002/138207617-dbae526e-04d6-4bf0-b0a1-5bc12ec0978f.png)|After submitting, the learner is brought back to the lesson, the score is shown, and the typical number of points is updated|
|![Screen Shot 2021-10-20 at 8 25 03 PM](https://user-images.githubusercontent.com/13563002/138207621-30d5aafb-7be8-4690-b906-2699c29c7b45.png)|Clicking on the quiz in the channel now allows learner to see their report|

### Notes and changes to current designs:
- [Figma for finding an independent quiz in a lesson](https://www.figma.com/file/TfaSTT9gs6fPKhGdNUjffU/Channel-quizzes?node-id=882%3A9622)
- There are three shared components that will be available for use, `PracticeQuizRenderer`, `ReportsPracticeQuizLearnerPage`, `PracticeQuizTimer`
- All strings required should already be in `commonCoreStrings`, `commonCoachStrings`, or `commonLearnStrings`
- Routes need to be created

### Criteria for success:
**Please copy these into your PR for "Steps for Testing"!**

This issue is done when the checkboxes and Gherkin stories below work
- [ ] Learner earns points after submitting the independent practice quiz and returning back to the lesson resource page 
- [ ] Learners can discover independent practice quizzes in Learn in assigned lesson
```
Feature: Learners can discover independent practice quizzes in Learn
	Scenario: Independent practice quizzes appear in assigned lessons
		Given that there is a lesson with an independent practice quiz assigned to my class
		When I click on the lesson card in my classes tab
		Then I see a list of ordered lesson resources which includes my independent practice quiz
```
- [ ] Learners can see time information while taking an independent practice quiz in lesson or channel
```
Feature: Learners can see time information while taking an independent practice quiz in lesson or channel	
	Scenario: Learners see how long they are taking on the independent practice quiz
		Given that I have entered the independent practice quiz page
		When the I start on the independent practice quiz
		Then I can see a time displayed as 'Time spent' above the question list
			And I can see that time increment by minutes

	Scenario: Learners see the suggested time to take on the independent practice quiz
		Given that I have entered the independent practice quiz page
		When the I start on the independent practice quiz
		Then I see a time displayed as 'Quiz duration' above the question list
			And I do not see that time increment
```
- [ ] Learners can see independent practice quiz report from independent practice quiz
```
Feature: Learners can see independent practice quiz reports from independent practice quizzes assigned in lessons or in channels
	Scenario: learner views independent practice quiz report after finishing the independent practice quiz
		Given that I have finished and submitted the independent practice quiz
			And I am back on in the channels tab
			And I see my score on the same independent practice quiz content card
		When I click on the content card
		Then I see my independent practice quiz report
			And I see my score and time information
			And I see a button to retake the independent practice quiz
```

**Comment by bjester** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Kolibri-Ecosystem-6047e525b8b9d900182e2d6f/issues/learningequality/kolibri/8508?planning-poker) with ZenHub @jredrejo @marcellamaki @sairina @rtibbles @MisRob @nucleogenesis @jamalex

**Comment by rtibbles** [comment]
Fixed in #8645
